core/shared/src/main/scala/zio/scheduler/Scheduler.scala

```diff
@@ -16,9 +16,31 @@
 
 package zio.scheduler
 
-import zio.{ Has, ZLayer }
-import zio.internal.{ Scheduler => IScheduler }
+import zio.{ Has, UIO, ZLayer }
+import zio.duration.Duration
+import zio.internal.Scheduler.CancelToken
 
 object Scheduler extends PlatformSpecific {
-  val live: ZLayer.NoDeps[Nothing, Has[IScheduler]] = ZLayer.succeed(globalScheduler)
+
+  trait Service extends Serializable {
+    def submit(task: Runnable, duration: Duration): UIO[Unit] =
```

Comment:
We can make this more idiomatic as `submit[A](task: => A, duration: Duration): UIO[A]`.

Reply:
Done.